.github/workflows/codeql.yml

```
@@ -0,0 +1,68 @@
+---
+# vi: ts=2 sw=2 et:
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+name: "CodeQL"
+
+on:
+  pull_request:
+    branches:
+      - master
+      - rhel-*
```

Wouldn'T be better: :smile: 
```suggestion
      - rhel-8.*.0
```

---

That's unnecessarily specific for no real gain here, imo.